README.md

```diff
@@ -41,50 +41,62 @@ qldb --ledger <your-ledger>
 
 ### Configuration
 
-After installation, the Shell will load default config file located at `$XDG_CONFIG_HOME/qldbshell/default_config.toml` during initialization. For example on OSX this will typically be at `/Users/username/Library/Application Support/qldbshell/default_config.toml`. You will need to create the `default_config.toml` config file manually after installation. The config file uses [TOML syntax](https://github.com/toml-lang/toml) and the format is as follows:
-
+After installation, the Shell will load default config file located at
+`$XDG_CONFIG_HOME/qldbshell/config.ion` during initialization. For example on
+OSX this will typically be at `~/Library/Application Support/qldbshell/config.ion`. If such a file doesn't exist, the shell will run
```

Comment:
i see that this PR also updates the macos config location in addition to changing to ion -- if that is the case, this should be updated (and maybe should specify our intentional decision to override?)